src/components/TabBar/TabBar.css

```diff
@@ -41,11 +41,4 @@
     border-color: var(--primaryLinkColor);
     color: color(var(--colorMuted) lightness(85%));
   }
-
-  @media only screen and (max-width: 1439px) {
-    white-space: nowrap;
```

Comment:
Honestly I'm not that big of a fan of the scrolling on mobile either.  While it's better than on desktop since the scrollbar isn't as large, the fact that it scroll isn't immediately obvious.

Currently with the 13/14 elements we have this breaks onto 4 lines (about 1/3 screen height).  I tested it in the Chrome mobile view and it looks OK to me.